docs/.vuepress/config.js

```diff
@@ -181,6 +191,45 @@ module.exports = {
           children: [""],
         },
       ],
+      "/enterprise/": [
+        {
+          title: "Enterprise",
+          type: "group",
+          collapsable: false,
+          sidebarDepth: 2,
+          //path: "/enterprise/",
```

Comment:
nit: trailing comments

Reply:
Yea, definitely need to go before merge. That's my WiP method, in case I break ALL THE THINGS and need to undo, without the commit history memorializing my shameful JS methods.